### PR TITLE
Release version 4.3.0 

### PR DESCRIPTION
**Pull request by guialbuk**
### Features

* Add `have_rich_text` matcher for `ActionText` ([#1263])

### Improvements
* Use range on `validate_exclusion_of#in_range` documentation ([#1273])

### Bug fixes

* Fix `missing attribute:: scope 1` intermittent test:  ([#1274])

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# 4.3.0
+
+### Features
+
+* Add `have_rich_text` matcher for `ActionText` ([#1263])
```
Comment (mcmire) [inline NEWS.md]: Do issue numbers get autolinked in Markdown documents? If so then you can remove the `[...]`, but if not, what do you think about adding a URL for this, a la:

```
[#1106]: https://github.com/thoughtbot/shoulda-matchers/pull/1106
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
-# Shoulda Matchers [![Gem Version][version-badge]][rubygems] [![Build Status][travis-badge]][travis] ![Downloads][downloads-badge] [![Hound][hound-badge]][hound]
+# Shoulda Matchers
+[![Gem Version][version-badge]][rubygems]
```
Comment (mcmire) [inline README.md]: This places all of the badges after the header, is that intentional? Fine if so, but just checking.

File: NEWS.md (hunk 1 above)
Comment (guialbuk) [reply NEWS.md]: Good point. They only get linked auto-linked in GitHub, so I added the links so it works anywhere else.

**Comment by mcmire** [comment]
Oops I forgot you had this open! I'll resist from merging in any more stuff.

**Comment by guialbuk** [comment]
@mcmire No worries 🙂
I'm just waiting for the tests to pass to release 4.3.0 🚀

**Pull request merged**